tests/src/test/scala/cats/tests/KleisliSuite.scala

```diff
@@ -195,6 +196,12 @@ class KleisliSuite extends CatsSuite {
     checkAll("MonoidK[Kleisli[Option, String, ?]]", SerializableTests.serializable(monoidk))
   }
 
+  {
+    implicit val coflatmap = Kleisli.catsDataCoflatMapForKleisli[Option, String]
```

Comment:
you shouldn't need this explicit instance. did it fail to compile without it?

Reply:
You're right - I've removed it.